Lil-Peep/src/styles/modules/featured-categories.scss

```diff
@@ -120,15 +126,17 @@
 
 @media only #{ $tablet-view } {
   .categories-main {
+    background-color:  $color-bg;
+    border-bottom: 1px solid  $color-bd;
     border-color: blue;
+    border-radius: 3px;
     border-style: solid;
     border: 5px;
-    border-radius: 3px;
-    background-color:  $color-bg;
-    border-bottom: 1px solid  $color-bd;
-    font-family:  $Lato;
     display: flex;
+    font-family:  $Lato;
```

Comment:
Name of variable `Lato` should be written in all lowercase letters with hyphens instead of underscores<br>Colon after property should be followed by one space